wiki/Beatmap_Editor_Guides/Changing_the_Map's_Artist_or_Title/en.md

```diff
@@ -26,9 +29,9 @@ Hard Method (keep local changes):
 4.  Go into editor. Make sure the "Web" menu is enabled (not greyed out).
 5.  Change the Artist/Title info.
 6.  **Do NOT exit the editor.**
-7.  Use **File &gt; Open...** to change the Artist/Title in other difficulties.
+7.  Use **File > Open...** to change the Artist/Title in other difficulties.
```

Comment:
idk, but since these are buttons, ``6.  Use `File` > `Open...` to change the Artist/Title in other difficulties.``